instruqt/distributing-with-replicated/02-enabling-the-sdk/assignment.md

```diff
@@ -0,0 +1,99 @@
+---
+slug: enabling-the-sdk
+id: k2xalui3fpuw
+type: challenge
+title: Enabling the Replicated SDK
+teaser: Incorporate the SDK into your application
+notes:
+- type: text
+  contents: Intoducing the Replicated SDK into your application
+tabs:
+- title: Shell
+  type: terminal
+  hostname: shell
+- title: Manifest Editor
+  type: code
+  hostname: shell
+  path: /home/replicant
+difficulty: basic
+timelimit: 300
+---
+
+Now that we've got our environment set up, let's incorporate the
+SDK into our application. We make it easy for you to do this by
+providing a Helm chart you can drop into your chart as a
+dependency. When you deliver your Helm chart from the Replicated
+regisry, we'll embed your customer's license into the final
+chart.
+
+This injection serves a few purposes:
+
+1. The licese is available to your application logic through a
+   call to an in-cluster SDK
```

Comment:
"call to an SDK" is not clear to me what's happening. I guess long term we'll provide libraries for querying the SDK but for now it's a rest call, and "call to an SDK" sounds like a function call.

how about:

> in-cluster SDK or HTTP API

Reply:
in cluster API makes more sense